open-webui/umbrel-app.yml

```diff
@@ -0,0 +1,33 @@
+manifestVersion: 1
+id: open-webui
+name: Open WebUI
+tagline: User-friendly AI Interface
+category: ai
+version: "0.4.8"
+port: 2876
+description: >-
+  🌐 Open WebUI is an extensible, feature-rich, and user-friendly self-hosted WebUI designed to operate entirely offline.
+
+
+  🤖 It supports various LLM runners, including Ollama and OpenAI-compatible APIs.
+
+
+  🦙 This app comes packaged with [Ollama](https://ollama.com/) pre-configured to work out of the box. 
+  
+  To start downloading models via Ollama, simply type in a model name in the model search bar and click "Pull from Ollama.com".
+  A list of models can be found at https://ollama.com/search.
```

Comment:
I've added some instructions for downloading models. We could also consider not doing this and instead linking to something like https://docs.openwebui.com/getting-started/quick-start/starting-with-ollama#a-quick-and-efficient-way-to-download-models